### PR TITLE
Add ethereum_spent_over_time metric

### PR DESCRIPTION
**Pull request by spiderjako**

### CODE REVIEW

File: san/sanbase_graphql.py [hunk 1]
```diff
@@ -95,25 +95,12 @@
...
-        'return_fields': ['balance', 'datetime']
-    },
-    'social_dominance':{
-        'query': 'socialDominance',
-        'return_fields': ['datetime', 'dominance']
-    },
-    'top_holders_percent_of_total_supply':{
-        'query': 'topHoldersPercentOfTotalSupply',
-        'return_fields': ['datetime', 'inExchanges', 'inTopHoldersTotal', 'outsideExchanges']
-    },
```
Comment (IvanIvanoff) [inline san/sanbase_graphql.py]: Why are these deleted?

File: san/sanbase_graphql.py [hunk 2]
```diff
@@ -280,7 +267,7 @@ def top_holders_percent_of_total_supply(idx, slug, **kwargs):
         idx=idx,
         slug=slug,
         **kwargs
-    )
+)
```
Comment (IvanIvanoff) [inline san/sanbase_graphql.py]: this indentation was correct

File: san/sanbase_graphql.py (hunk 1 above)
Comment (spiderjako) [reply san/sanbase_graphql.py]: Because they're not "equal format" functions, that means that they're not created via "_create_query_string", which automates the process and grabs the data for making functions from the QUERY_MAPPING

File: san/sanbase_graphql.py (hunk 2 above)
Comment (spiderjako) [reply san/sanbase_graphql.py]: Yes, you're right, fix is on the way

File: README.md [hunk 3]
```diff
@@ -1136,14 +1137,36 @@ Example result:
...
 2019-05-17 09:02:07+00:00  Debt-Ridden Crypto Exchange Cryptopia Suckers ...  Debt-Ridden Crypto Exchange Cryptopia Suckers ...              CCN                  https://www.ccn.com/debt-ridden-crypto-exchange-cryptopia-suckers-hacked-customers-again
 2019-05-27 18:56:15+00:00  Institutions Could Push Crypto Past A ‘Point O...  Institutions Could Push Crypto Past A ‘Point O...  Crypto Briefing                                          https://cryptobriefing.com/institutional-crypto-point-no-return/
 2019-06-22 14:31:00+00:00  ETH Hits 10-Month High as Crypto Markets See S...  ETH Hits 10-Month High as Crypto Markets See S...    Cointelegraph                   https://cointelegraph.com/news/eth-hits-10-month-high-as-crypto-markets-see-solid-green
 2019-07-06 18:42:00+00:00  Iranian Official Says US Congress is Working t...  Iranian Official Says US Congress is Working t...    Cointelegraph  https://cointelegraph.com/news/iranian-official-says-us-congress-is-working-to-block-irans-crypto-mining
+```
+
+### Ethereum Spent Over Time
+
+ETH spent for each interval from the project's team wallet and time period
```
Comment (IvanIvanoff) [inline README.md]:
```suggestion
ETH spent for each interval from the project's team wallet(s) and time period
```
